pages/docs/community/styleguide/inclusive-language.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Inclusive Language style guide
+description: This style guide outlines the guideline for using inclusive languages in documentation.
+---
+
+# Inclusive Language
+
+While writing documentation we are indirectly reaching out to thousands of people all around the globe. 
+Thus, we must ensure that our diverse audience can connect with our information. 
+
+### 1. Culturally Inclusive Language
+
+Some phrases and words are popularly used only in specific regions of the world, thus our audience from other parts 
+of the world might need help to relate to the same. Hence, we should try to avoid using such phrases in our documentation. 
+Some examples of Region Specific Language are -
+- “It’s not rocket science” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+- Phrases written in regional languages instead of English.
+- “It’s not a piece of cake” is commonly used in the USA, and non-US people might not be able to relate to it properly.
+
+### 2. Gender Neutral Language - 
+
+Some phrases and words are targeting men or men/women specifically. This might result in other gender groups feeling 
+left out while reading our documentation. 
+Some examples of Gender Neutral Language are -
+- Using “they/them” instead of “his/him” or “she/her”.
+- Using “Hello everyone” instead of “Hello guys”.
+- Using “Chairperson” instead of “Chairman” or “Chairwoman”.
+
+### 3. People First Language - 
+
+The placement of the word “People” matters a lot when it comes to writing documentation. 
+Using the word “People” in the beginning makes it more pleasant as the problem is not displayed in the beginning.
+Some examples of Gender Neutral Language are -
+- Using “People with Mental Instability” instead of “Mentally Unstable People”.
+- Using “People with Audio Impairment” instead of “Deaf People”.
+
+### 4. Slang Free Language -
```

Comment:
This section is titled `slang`, but focuses on `vulgar language`. 

**Slang and vulgar language are not the same**, although they certainly may overlap in some cases. _Slang refers to informal words or expressions that are often used in specific communities or groups, and it can be considered more casual or colloquial than standard language._ (Slang terms are not inherently vulgar or offensive.)

On the other hand, _vulgar language refers to language that is considered crude, obscene, or offensive._ It often includes swear words, sexually explicit language, and may be considered inappropriate in certain contexts or situations. **Vulgar language can be slang, but not all slang is vulgar.**


**ACTION ITEMS:** 
1. Please fix this heading and re-write it to actually focus on `slang` topic with dedicated `slang` examples. 
2. Please add an additional section and header for `vulgar`, which is it's own topic that deserves it's own examples. Please remember to be cautious what examples you add to this section and be mindful to select examples that do not make the community feel triggered or offended.

Reply:
Here are some examples of incorrect slang terms and corrected alternatives to help you get started! ✨

**Incorrect Slang Terms:**
- "Crazy" - using this term to describe something unusual or unexpected is stigmatizing and offensive to people with mental health conditions.
- "Lame" - using this term to describe something boring or uncool is hurtful to people with disabilities.
- "Gay" - using this term as an insult or to describe something negative or undesirable is homophobic and offensive to the LGBTQ+ community.
- "Redneck" - using this term to describe someone from a rural area can be pejorative and offensive.
- "Ghetto" - using this term to describe something low-quality or undesirable can be stigmatizing and offensive to people living in poverty or urban areas.

**Corrected Alternatives:**
- "Unusual," "unexpected," or "surprising"
- "Boring," "uncool," or "uninteresting"
- Use language that is not insulting or offensive to any particular group. For example, instead of saying, "that's so gay," you could say, "that's not my thing," or "that's not cool."
- "Rural," "country," or "small town"
- "Low-quality," "undesirable," or "run-down"



_source: This is the edited version of the query reply I received from ChatGPT when I asked it for 5 ideas of incorrect slang terms and corrected alternatives._